homeassistant/components/binary_sensor/mqtt.py

```diff
@@ -29,8 +30,8 @@
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Optional(CONF_PAYLOAD_OFF, default=DEFAULT_PAYLOAD_OFF): cv.string,
     vol.Optional(CONF_PAYLOAD_ON, default=DEFAULT_PAYLOAD_ON): cv.string,
-    vol.Optional(CONF_SENSOR_CLASS, default=None):
-        vol.Any(vol.In(SENSOR_CLASSES), vol.SetTo(None)),
```

Comment:
That's weird, we shouldn't. I have never seen `SetTo` before. (quick grep shows this is the only place it is used)